### PR TITLE
Io 5919 add asset id update

### PR DESCRIPTION
**Pull request by GimmeYerGold**
Corrected how to update an asset with a PUT request; clarified how PUTs work with resource IDs in the main section.

https://ioffice.myjetbrains.com/youtrack/issue/io-5919

### CODE REVIEW

File: api/endpoints/asset.md [hunk 1]
```diff
@@ -39,11 +39,10 @@ To update a Asset send JSON with updated value for one or more of the attributes
     
 + Request (application/json)
 
-            {
-                "model": {
-                    "id": 78
-                }
-            }
+			{
```
Comment (quesojean) [inline api/endpoints/asset.md:45]: might as well do this for all of the endpoints PUT calls (adding the id in the payload) - reservation, user, etc.

**Review by quesojean — CHANGES_REQUESTED** [review]
dont kill me, but to be less confusing to readers, it would better to amend every resource's PUT payload to include the id.

File: api/endpoints/asset.md (hunk 1 above)
Comment (GimmeYerGold) [reply api/endpoints/asset.md:45]: @quesojean ✅ added those 👍

File: api/endpoints/agreement.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ A single agreement object.
     [Agreement][]
 
 ### Edit a Agreement [PUT]
-To update a Agreement send JSON with the id and updated value for one or more of the attributes.
+To update a Agreement send JSON with the Agreement's `ID` and updated value for one or more of the attributes.
+
++ Parameters
+    + id (string) ... ID of the Agreement
     
 + Request (application/json)
```
Comment (quesojean) [inline api/endpoints/agreement.md:33]: so i just confirmed with @kentongray , model id only needed for POST request in asset.  you mind updating the docs for POST? (create)

**Review by quesojean — CHANGES_REQUESTED** [review]

File: api/endpoints/agreement.md (hunk 2 above)
Comment (GimmeYerGold) [reply api/endpoints/agreement.md:33]: @quesojean looks like you commented about `asset` on the `agreement` section of the docs, but I did add those changes.

File: api/api.md [hunk 3]
```diff
@@ -42,15 +42,25 @@ Example usage to search users by employeeID:
...
-    e.g. Updating a resource's contact to another with an ID of 5
-    + Request (application/json)
-        {
-            contact: {
-                id: 5
-            }
-        }
+* e.g. Updating a resource's `printerName` to another, when that resource's `id` is `5`.
+
+Headers
```
Comment (jmlopez-rod) [inline api/api.md:51]: Perhaps give a sample url, usually the endpoints require the id in the url as well. Some, not all.

File: api/endpoints/asset.md [hunk 4]
```diff
@@ -35,15 +35,17 @@ A single asset object.
     [Asset][]
 
 ### Edit a Asset [PUT]
-To update a Asset send JSON with updated value for one or more of the attributes.
+To update a Asset send JSON with the Asset's `ID` and updated value for one or more of the attributes.
```
Comment (jmlopez-rod) [inline api/endpoints/asset.md]: to update `an` Asset.

**Review by jmlopez-rod — CHANGES_REQUESTED** [review]
Two typos

File: api/api.md (hunk 3 above)
Comment (GimmeYerGold) [reply api/api.md:51]: @jmlopez-rod when this section is compiled, a url example is shown. Does this suffice? 
image

Comment (jmlopez-rod) [reply api/api.md:51]: Yes, its enough

**Review by jmlopez-rod — APPROVED** [review]

**Review by quesojean — APPROVED** [review]

**Pull request merged**